src/app/(nextjs_migration)/(guest)/user/unsubscribe-monthly/page.tsx

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { redirect } from "next/navigation";
+import { updateMonthlyEmailOptout } from "../../../../../db/tables/subscribers";
+import { getL10n } from "../../../../functions/server/l10n";
+
+export default async function PublicScan(props: {
+  searchParams: { [key: string]: string | string[] | undefined };
+}) {
+  const token = props.searchParams.token;
+  if (typeof token !== "string") {
+    return redirect("/");
+  }
+  try {
+    await updateMonthlyEmailOptout(token);
+  } catch (error) {
+    console.log(error);
+    return redirect("/");
+  }
+  const l10n = getL10n();
+
+  return (
+    <div data-partial="allBreaches">
```

Comment:
Question (non-blocking): Is `data-partial="allBreaches"` needed here?

Reply:
Oops. Probably not needed, but to introduce as little changes as possible: e10c7897f